src/components/Container.js

```diff
@@ -32,11 +33,32 @@ const StyledContent = styled(motion.div)`
   display: flex;
   flex-direction: column;
   z-index: 99999;
+  position: relative;
 `
 
 const StyledMedian = styled(motion.div)`
   ${props => props.padded && `padding: 0 ${spacing.padding.small}px`};
-  margin-top: -${spacing.padding.small}px;
+  margin-top: -${forms.input.height.regular / 2}px;
+  position: relative;
+  &:after {
+    content: '';
+    display: ${props => (props.gradient ? 'block' : 'none')};
+    position: absolute;
+    bottom: -13px;
+    left: 0;
+    width: 100%;
+    height: 10px;
+    border-top: 3px solid ${props => colors[props.gradientColor]};
+    background: linear-gradient(
+      ${props => colors[props.gradientColor]},
+      ${props =>
+        Color(colors[props.gradientColor])
+          .fade(1)
+          .rgb()
+          .string()}
+    );
+    z-index: 9999999;
```

Comment:
Why would we need z-index here? The widget has an option to override a z-index, would this conflict then?

Reply:
This is specifically for the median, which is the container for the search bar